dags/openshift_nightlies/scripts/run_scale-ci-diagnosis.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
can this script be moved into a a dir `dags/openshift_nightlies/scripts/utils/` and rename the file to use either _ or - but not both?

Reply:
done